fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -142,6 +142,7 @@ def __init__(
 
         self.buckets: Dict[OSS, Dict[torch.device, List[Bucket]]] = {o: {} for o in self.sharded_optimizers}
         self._should_bucket_grad: List[bool] = []
+        self._bucket_list: Optional[List[Bucket]] = None
```

Comment:
add a bit comment on each of the class local variable to explain what do they do and the purpose and benefits?